app/src/main/scala/uk/ac/wellcome/platformapi/controllers/MainController.scala

```diff
@@ -0,0 +1,63 @@
+package uk.ac.wellcome.platform.api.controllers
+
+import com.twitter.finagle.http.Request
+import com.twitter.finatra.http.Controller
+import uk.ac.wellcome.platform.api.services._
+
+import javax.inject.{Inject, Singleton}
+import com.sksamuel.elastic4s.ElasticDsl._
+import scala.concurrent.Future
+import org.elasticsearch.action.admin.cluster.health.ClusterHealthResponse
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.util.Try
+
+import com.twitter.finatra.request.QueryParam
+import com.twitter.finatra.validation.NotEmpty
+
+import uk.ac.wellcome.platform.api.models._
+
+
+case class CalmRequest(
+  @NotEmpty @QueryParam altRefNo: String
+)
+
+case class RecordCollectionPair(
+  record: Option[Record],
+  collection: Option[Collection]
+)
+
+case class RecordResponse(
+  altRefNo: String,
+  entries: Record,
+  parent: Option[Collection]
+)
+
+@Singleton
+class MainController @Inject()(
+  calmService: CalmService
+) extends Controller {
+
+  val apiBaseUrl = "/api/v0"
```

Comment:
Not that it’s hugely important right now, but I think the consensus was we drop the `/api` from URLs, as it’s all being served from `api.wc.org`.